facia-tool/app/LogStash.scala

```diff
@@ -0,0 +1,114 @@
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.classic.{Logger => LogbackLogger, LoggerContext}
+import ch.qos.logback.core.FileAppender
+import ch.qos.logback.core.util.Duration
+import com.gu.logback.appender.kinesis.KinesisAppender
+import conf.Configuration
+import net.logstash.logback.appender.LogstashTcpSocketAppender
+import net.logstash.logback.encoder.LogstashEncoder
+import net.logstash.logback.layout.LogstashLayout
+import org.slf4j.{Logger, LoggerFactory}
+import play.api.{Logger => PlayLogger, LoggerLike}
+
+object LogStash {
+
+  lazy val loggingContext = LoggerFactory.getILoggerFactory.asInstanceOf[LoggerContext]
+  import play.api.Play.current
+
+  val config = play.api.Play.configuration
+
+  case class KinesisAppenderConfig(stream: String, region: String, roleArn: String, bufferSize: Int)
+
+  lazy val enabled = config.getBoolean("logging.logstash.enabled").getOrElse(false)
```

Comment:
It looks nicer when all configuration variables are defined in a single file, `configuration.scala`